scripts/screeps_rl_training_runner.py

```
@@ -336,6 +339,70 @@ def validate_card_supply(card: JsonObject) -> None:
             raise TrainingCardError("consumed Loop A card supply requires consumed_by_report_id")
 
 
+def validate_scenario_metadata(card: JsonObject) -> None:
```

Ruff (0.15.12)

[warning] Too many branches (18 > 12)

(PLR0912)